claim/models.py

```diff
@@ -337,6 +308,38 @@ def get_queryset(cls, queryset, user):
                 if not isinstance(user._u, core_models.TechnicalUser):
                     queryset = LocationManager().build_user_location_filter_query( user._u, prefix='health_facility__location', queryset = queryset, loc_types=['D'])
         return queryset
+      
+class FeedbackPrompt(core_models.VersionedModel):
+    id = models.AutoField(db_column='FeedbackPromptID', primary_key=True)
+    feedback_prompt_date = fields.DateField(db_column='FeedbackPromptDate', blank=True, null=True)
+    claim = models.ForeignKey(
+        Claim, models.DO_NOTHING, db_column='ClaimID', blank=True, null=True, related_name="+")   
+    officer = models.ForeignKey(
+        core_models.Officer , models.DO_NOTHING, db_column="OfficerID", blank=True, null=True)
+    phone_number = models.CharField(db_column='PhoneNumber', max_length=50)
+    sms_status = models.IntegerField(db_column='SMSStatus', blank=True, null=True)
+    validity_from = fields.DateTimeField(db_column='ValidityFrom', blank=True, null=True)
+    validity_to = fields.DateTimeField(db_column='ValidityTo', blank=True, null=True)
+    legacy_id = models.IntegerField(db_column='LegacyID', blank=True, null=True)
+    audit_user_id = models.IntegerField(db_column='AuditUserID', blank=True, null=True)
```

Comment:
Is there any change there, or those lines were just moved?

Reply:
Moved to allow the FK to claim (claim was defined after ... )